aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -170,52 +188,44 @@ namespace native {
 
 DEFINE_DISPATCH(addr_stub);
 
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ linalg.det ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
How do you know how many ~ to put? 😄

Reply:
I literally eyeball them, although I'm sure there's a vim plugin for this lol